### ISSUE TITLE
Feature: Allow Posting it as Comment

### ISSUE THREAD

**Issue by rowi1de** [flowwer-dev/pull-request-stats#24]
Please add a feature that allows switching from Description to Comment.

```yaml

   
name: Pull Request Stats

on: 
  pull_request:
    types: [opened]
  

jobs:
  stats:
    runs-on: ubuntu-latest
    steps:
      - name: Run pull request stats
        uses: flowwer-dev/pull-request-stats@master
        with:
          comment: true # set trigger accordingly, default is opened, synchronize, and reopened
```

**Comment by rowi1de** [comment]
Same as https://github.com/flowwer-dev/pull-request-stats/issues/14

**Comment by manuelmhtr** [comment]
Hi @rowi1de. You are right, commenting works when configuring the trigger correctly. I'll include this option in the next version. Let me find some time to work on that.

**Comment by DaniAkash** [comment]
@manuelmhtr when the PR body is edited - the unfurling of pr links prints the users as preview image. This makes the PR previews when shared on slack awkward

**Comment by manuelmhtr** [comment]
Hi @DaniAkash, I see. I make posting as comment makes ever more sense. Let me do some time this week to launch this feature.

**Comment by manuelmhtr** [comment]
Hi @rowi1de @DaniAkash, publishing as comment is now available in [v2.4.0](https://github.com/flowwer-dev/pull-request-stats/releases/tag/v2.4.0). It's now the default behavior and configurable with the `publish-as` option.
